omnigibson/prims/entity_prim.py

```diff
@@ -1531,18 +1567,28 @@ def create_attachment_point_link(self):
     def _dump_state(self):
         # We don't call super, instead, this state is simply the root link state and all joint states
         state = dict(root_link=self.root_link._dump_state())
-        joint_state = dict()
-        for prim_name, prim in self._joints.items():
-            joint_state[prim_name] = prim._dump_state()
-        state["joints"] = joint_state
+        if self.n_joints > 0:
+            state["joint_pos"] = self.get_joint_positions()
+            state["joint_vel"] = self.get_joint_velocities()
+            state["joint_eff"] = self.get_joint_efforts()
+            # state["joint_pos_target"] = self.get_joint_position_targets()
+            # state["joint_vel_target"] = self.get_joint_velocity_targets()
```

Comment:
still needed?

Reply:
Maybe add a comment to explain why we explicitly DO NOT save/load state for joint targets

Reply:
done